examples/flutter_gallery/lib/demo/animation/home.dart

```diff
@@ -584,7 +592,8 @@ class _AnimationDemoHomeState extends State<AnimationDemoHome> {
                     height: 610.0,
                     child: NotificationListener<ScrollNotification>(
                       onNotification: (ScrollNotification notification) {
-                        return _handlePageNotification(notification, _detailsPageController, _headingPageController);
+                        return _handlePageNotification(
+                          notification, _detailsPageController, _headingPageController);
```

Comment:
Here and elsewhere: the line width limit needs to be softer. Adding line breaks here just makes the code look we're trying to cram it into newspaper columns.